src/libs/actions/PersonalDetails.js

```diff
@@ -135,7 +122,7 @@ function fetch() {
             myPersonalDetails.lastName = lodashGet(data.personalDetailsList, [currentUserEmail, 'lastName'], '');
 
             // Set my personal details so they can be easily accessed and subscribed to on their own key
-            Onyx.merge(ONYXKEYS.MY_PERSONAL_DETAILS, myPersonalDetails);
+            Onyx.set(ONYXKEYS.MY_PERSONAL_DETAILS, myPersonalDetails);
```

Comment:
Similar question about why set and not merge. But I can't think of any place we are modifying the `myPersonalDetails` I think probably after the timezone changes in this PR we are just always overwriting the details with those from the server on page refresh. Which seems fine?